shared/components/map.js

```diff
@@ -81,6 +83,7 @@ export default class Map extends Component {
       buttonTranslateY: new Animated.Value(0),
       userLocationEnabled: false,
       itemLength: 0,
+      itemCategory: '',

```

Comment:
oh no, they are different. categoryFilter is from tabview in main header and this itemCategory is from the marker a user pressed